WalletWasabi.Tests/Helpers/MemPoolInfoGenerator.cs

```diff
@@ -0,0 +1,60 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using NBitcoin;
+
+namespace WalletWasabi.Tests.Helpers
+{
+	public class MemPoolInfoGenerator
```

Comment:
```suggestion
	public class MempoolInfoGenerator
```

I think "mempool" is the correct word.

Reply:
Done.